sympy/sets/tests/test_sets.py

```diff
@@ -1500,6 +1500,12 @@ def test_DisjointUnion():
     assert DisjointUnion(Interval(-oo, oo)).rewrite(Union) == Interval(-oo, oo) * FiniteSet(0)
     assert DisjointUnion(S.EmptySet).rewrite(Union) == S.EmptySet
     assert DisjointUnion().rewrite(Union) == S.EmptySet
+    error_raised = False
+    try:
+        DisjointUnion(Symbol('n'))
+    except ValueError:
```

Comment:
You should better use ‘raises’ from sympy.utilities.pytest

Reply:
Done.